docs/installation/octopus-in-container/octopus-server-container-windows.md

```diff
@@ -1,22 +1,19 @@
 ---
-title: Octopus Deploy Server Container Windows
+title: Octopus Deploy Server Windows Container
 description: An Octopus Deploy Server instance can be run directly from within a container.
 position: 1
 ---
 
 If you want to run an Octopus Deploy Linux container, please refer to the [Octopus Deploy Server Container Linux](/docs/installation/octopus-in-container/octopus-server-container-linux.md) documentation.
 
+Note: When running Windows Containers, please ensure you have Docker set to [use Windows Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
+
 Running the Octopus Deploy Server inside a container provides a simple way to set up an Octopus Deploy instance. Upgrading to the latest version of Octopus is just a matter of running a new container with the new image version.
 
 Although there are a few different configuration options, a simple example of starting up an Octopus Deploy server container is as follows:
 
 ```PowerShell
-docker run --interactive 
-           --detach `
-           --name OctopusServer `
-           --publish 1322:81 `
-           --env DB_CONNECTION_STRING="..." `
-           octopusdeploy/octopusdeploy:2019.13.4
+docker run --interactive --detach --name OctopusServer --publish 80:8080 --env DB_CONNECTION_STRING="..." --env ACCEPT_EULA="Y" --volume "C:/Octopus/Data:C:/Octopus" octopusdeploy/octopusdeploy:2019.13.4
```

Comment:
Doesn't the container listen on 81 internally?

Reply:
It listens on 8080 - Will change host port to higher port number